### PR TITLE
GP2-1915 Align EP data snapshots

### PR DESCRIPTION
**Pull request by bobmeredith**
This PR changes the export plan data snapshots to use the same data mechanisms as WTE and aligns the formatting.
While the WTE page uses a single transparent endpoint that pipes directly though to directory-api, the Export plan has a view that collects all data required for display using a single endpoint on directory-api.  Filtering based on target age groups is also carried out by this view.  Initial population on the Marketing approach page has been removed since this was immediately replaced by an ajax request. The performance hit of double-loading is therefore removed.
Given that the new endpoints are significantly faster than the old ones - a considerable increase in performance is expected from all data-liked EP pages.

Following this work,the entire old population model (which loads population data on-the-fly) can be removed from directory API, along with several other now redundant endpoints.

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/GP2-1915
- [x] Jira ticket has the correct status.
- [x] [Changelog](CHANGELOG.md) entry added.

### Reviewing help

- [ ] Explains how to test locally, including how to set up appropriate data
- [x] Includes screenshot(s) - ideally before and after, but at least after

### Housekeeping

- [ ] Added all new environment variables to Vault.
- [ ] Cleaned up old feature flags
- [ ] Upgraded any vulnerable dependencies.
- [ ] I have updated security dependencies
- [ ] Python requirements have been re-compiled.
- [x] Frontend assets have been re-compiled.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

**Comment by bobmeredith** [comment]
image
image

File: core/helpers.py [hunk 1]
```diff
@@ -29,29 +30,22 @@
...
-    '65-69',
-    '70-74',
-    '75-79',
-    '80-84',
-    '85-89',
-    '90-94',
-    '95-99',
-    '100+',
-]
+def age_group_mapping(target_ages):
```
Comment (depsiatwal) [inline core/helpers.py:33]: is it worth worth moving this to directory_constants to make choices consistent with other choices ?

**Review by depsiatwal — COMMENTED** [review]
Assume it's been tested on changes :-) 

LGTM to me generally

File: core/helpers.py (hunk 1 above)
Comment (bobmeredith) [reply core/helpers.py:33]: I'm not sure.  The only other place this 'mapping' is used seems to be in javaScript, so it's no use as a python fn.

File: react-components/src/views/sections/MarketingApproach/index.jsx [hunk 2]
```diff
@@ -1,18 +1,25 @@
...
 
+import { Provider } from 'react-redux'
+import Services from '@src/Services'
 import { RouteToMarket } from '@src/components/RouteToMarket'
 import { TargetAgeGroupInsights } from '@src/components/TargetAgeGroupInsights'
 import { FormElements } from '@src/components/FormElements'
 
 export const createRouteToMarket = ({ element, ...params }) => {
-  ReactDOM.render(<RouteToMarket {...params} />, element)
+	ReactDOM.render(<RouteToMarket {...params} />, element)
```
Comment (dahfool) [inline react-components/src/views/sections/MarketingApproach/index.jsx:11]: @bob I think it's 2 spacing unit.

**Review by depsiatwal — APPROVED** [review]
LGTM

File: react-components/src/components/TargetAgeGroupInsights/DemoData/index.jsx [hunk 3]
```diff
@@ -4,6 +4,15 @@ import { Tooltip } from '@components/tooltip/Tooltip'
 import { Stats } from '@src/components/Stats'
 import { notAvailable } from '@src/components/Stats/StatsGroup'
 import PropTypes from 'prop-types'
+import { millify, normaliseValues, get } from '@src/Helpers'
+
+const formatNumber = (val) => Number(val) ? millify(Number(val)*1000) : notAvailable
+
+const formatLanguages = (languages) => {
```
Comment (dahfool) [inline react-components/src/components/TargetAgeGroupInsights/DemoData/index.jsx:11]: Could `formatLanguages` in react-components/src/components/TargetAgeGroupInsights/utils.js be used.

Comment (bobmeredith) [reply react-components/src/components/TargetAgeGroupInsights/DemoData/index.jsx:11]: Good thought - I'll do that.

File: react-components/src/Helpers.js [hunk 4]
```diff
@@ -65,6 +65,10 @@ const normaliseValues = (str, places = 1, fixed = false) => {
   return 'Data not available'
 }
 
+const numberWithSign = (value) => {
```
Comment (dahfool) [inline react-components/src/Helpers.js:68]: numberWithSign might a default value or return 0 if no params is passed

File: react-components/src/views/sections/MarketingApproach/index.jsx (hunk 2 above)
Comment (bobmeredith) [reply react-components/src/views/sections/MarketingApproach/index.jsx]: Quite right - and I have no idea why my editor is doing this.  Just seems to be two files.

**Pull request merged**